tasks/install-with-package.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install Certbot.
+  package: name=certbot state=present
```

Comment:
Also to get the tests to pass you can change the travis setup to test the install from source scenario on centos6 (as there's no certbot package available anyways).

@geerlingguy recently made a quite [similar change in his `drush` role](https://github.com/geerlingguy/ansible-role-drush/commit/1fd265dc3c156a17233c8cb6dfcefdd4f43288c4). Have a look at the `.travis.yml` and `tests/test-source-install.yml` changes.